wazo_dird_client/client.py

```diff
@@ -2,10 +2,29 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from wazo_lib_rest_client.client import BaseClient
+import wazo_dird_client.commands
 
 
 class DirdClient(BaseClient):
     namespace = 'wazo_dird_client.commands'
 
+    backends: wazo_dird_client.commands.backends.BackendsCommand
```

Comment:
I don't understand why this is necessary. If you add the entry_points to the setup.py file all commands will be discovered by stevedore by the BaseClient class

Reply:
This is for typing and editor autocomplete. This has no runtime impact, but improves developer experience.

Reply:
The dynamic loading with stevedore hides the existence of those commands from any static analysis tooling(IDE, type checkers), which makes it harder to understand what you're dealing with when using this as library.